docs/release-notes/8.4.4-4.md

```diff
@@ -0,0 +1,71 @@
+# Percona XtraDB Cluster 8.4.4-4 (2025-03-)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
```

Comment:
The first sentence sounds like PXC only works in a cloud environment.

Reply:
updated